examples/contracts/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "contract-examples",
```

Could this be prefixes with @examples?

```suggestion
  "name": "@examples/contract-examples",
```